.circleci/cimodel/data/dimensions.py

```diff
@@ -8,8 +8,8 @@
 ]
 
 ROCM_VERSIONS = [
-    "3.9",
```

Comment:
i am not familiar with the policy. but do we want to keep this since it's a major version upgrade?

Reply:
Nightly wheel policy is same as CI image policy. Keep latest release version and one version prior.